src/OpenLoco/src/Windows/Vehicle.cpp

```diff
@@ -193,12 +193,13 @@ namespace OpenLoco::Ui::Windows::Vehicle
             orderSkip,
             orderDelete,
             orderUp,
-            orderDown
+            orderDown,
+            orderReverse
         };
 
         // 0x00500554
         static WindowEventList events;
-        constexpr uint64_t enabledWidgets = (1 << widx::routeList) | (1 << widx::orderForceUnload) | (1 << widx::orderWait) | (1 << widx::orderSkip) | (1 << widx::orderDelete) | (1 << widx::orderUp) | (1 << widx::orderDown) | Common::enabledWidgets;
+        constexpr uint64_t enabledWidgets = (1 << widx::routeList) | (1 << widx::orderForceUnload) | (1 << widx::orderWait) | (1 << widx::orderSkip) | (1 << widx::orderDelete) | (1 << widx::orderUp) | (1 << widx::orderDown) | (1 << widx::orderReverse) | Common::enabledWidgets;
```

Comment:
should really be using 1ULL for constructing a uint64_t. I know we don't do that consistently through the codebase though.